tendrl/commons/objects/cluster/atoms/stop_monitoring_services/__init__.py

```diff
@@ -72,6 +72,13 @@ def run(self):
                         job_id=self.parameters['job_id'],
                         flow_id=self.parameters['flow_id'],
                     )
+                    for child_job_id in child_job_ids:
+                        child_job = NS.tendrl.objects.Job(
+                            job_id=child_job_id
+                        ).load()
+                        if child_job.status not in ["finished", "failed"]:
+                            child_job.status = "failed"
```

Comment:
please add comment here explaining the status check and change

Reply:
Child jobs in Import and Unmanage remain in new state if parent job gets timed out and failed. Child jobs should be marked failed explicitly and should not be waited for ttl to mark them failed. These stale jobs are picked up by node-agent in a scenario in which an import is fired but one of the nodes is down. After import failure if the node comes up, it will pick up those jobs and keep the node-agent busy and defer other tasks such as Unmanage( if fired immediately after import failure) which may fail them due to timeout.

Reply:
I mean add code comment :)

Reply:
same case here

Reply:
okay.